index.html

```diff
@@ -947,620 +974,1301 @@
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-lg-12 px-0 subscribe-container border-top pt-4">
+            <div class="container px-0">
+              <div class="row">
+                <div class="col-lg-9 pr-4">
+                  <p class="subscribe-title">SUBSCRIBE</p>
+                  <div class="container background-gainsboro pr-2 mr-4">
+                    <div class="row">
+                      <div class="col-lg-5 py-5 pl-4">
+                        <img
+                          class="img-fluid"
+                          src="./assets/subscribe-newsletter-img.JPG"
+                          alt="newsletter subscription image"
+                        />
+                      </div>
+                      <div class="col-lg-7 py-5 mt-2">
+                        <strong class="subscribe-title-description">
+                          In-depth Stories You Don't See Elsewhere About Topics
+                          You Want to Read!
+                        </strong>
+                        <p class="subscribe-description mt-3">
+                          All Access + Weekly Delivery. Choose A Membership
+                          That's Perfect for You!
+                        </p>
+                        <button
+                          class="subscribe-button button-tag-magazine-page px-2 rounded"
+                        >
+                          SUBSCRIBE
+                        </button>
+                      </div>
+                    </div>
+                  </div>
                 </div>
-                <div class="text">
-                  "Unity starts with accountability," Warren said Sunday during
-                  an interview with CNN's Abby Phillip.
+                <div class="col-lg-3 pl-0">
+                  <p class="subscribe-title">NEWSLETTER</p>
+                  <div class="container pr-0">
+                    <div class="row">
+                      <div class="col-lg-6 newsletter-end-image px-0 py-0">
+                        <img
+                          src="./assets/newsweek-bottom-img.JPG"
+                          alt="Newsweek image phone"
+                          class="img-fluid newsweek-bottom-img"
+                        />
+                      </div>
+                      <div class="col-lg-6 border py-4 px-4">
+                        <strong class="newsletter-title">The Starting 5</strong>
+                        <p class="newsletter-description">
+                          See why nearly a quarter of a million subscribers
+                          begin their day with the Starting 5.
+                        </p>
+                        <p class="newsletter-email border px-3 py-1">
+                          Email address
+                        </p>
+                        <button class="newsletter-button">FREE SIGN UP</button>
+                      </div>
+                    </div>
+                  </div>

```

Comment:
- [ ] The Newsletter section layout is breaking on wide screens:
![Screenshot_2021-01-27 Bootstrap 4 Example](https://user-images.githubusercontent.com/16193212/106026231-54e50700-60a8-11eb-8147-06e06493f6dc.png)

Please fix it so it looks like the original version below:
![subscribe_and_newsletter_desktop](https://user-images.githubusercontent.com/16193212/106026293-67f7d700-60a8-11eb-86f2-9c708bd0171a.png)